Sage/carbon
# feat(full-screen-heading, heading): describe components using design tokens, update tests - FE-4930

### Proposed behaviour

Describes `FullScreenHeading` and `Heading components` using design tokens.
Updates tests when the above changes are made.

### Current behaviour

 `FullScreenHeading` and `Heading` components use the theme styled-component property.

### Checklist




#### QA


### Testing instructions

The correct appearance can be checked in the components: 
- `FullScreenHeading`: DialogFullScreen/Pages
- `Heading`
Use devtools to check if components use css variables.
Screenshots of tokens attached in the task.
